package.json

```diff
@@ -3,7 +3,7 @@
   "version": "3.0.3",
   "description": "Seedable random number generator supporting many common distributions.",
   "main": "dist/cjs/index.js",
-  "module": "dist/esm/index.js",
+  "module": "dist/esm/index.esm.js",
```

Comment:
not sure i see where it'll actually build `index.esm.js` as opposed to just `index.js`?

Reply:
It will build both of these index files in each outDir . However, `index.js` will not be referenced in a esm environment and `index.esm.js` will not be referenced in a commonjs environment due to the main and module values. To make this explicit I placed an `  "exclude": ["**/index.esm.ts"]` in tsconfig.csj.json with the same declaration for index.js in tsconfig.esm.json. Now each outDir will only contain the necessary index file. 

While my approach works I do not find it very elegant. I have researched this issue and I cannot find a consensus on a clean solution.